roles/ezid/tasks/main.yml

```diff
@@ -2,6 +2,5 @@
 # roles/ezid/tasks/main.yml
 - name: add ezid shared config
   become: yes
-  template: src=ezid.yml.j2 dest=/opt/{{ project_name }}/shared/config/ezid.yml owner={{ capistrano_user }} group={{ capistrano_group }} backup=no
-
-      
\ No newline at end of file
+  template: src=ezid.yml.j2 dest={{ config_path }}/ezid.yml \
```

Comment:
I'd prefer to use Ansible multiline format rather than backslashes for line continuation. You've done this in a number of places, though, so I'll file a ticket for that and we can address those all at once.